app/src/main/java/org/wikipedia/main/MainFragment.java

```diff
@@ -512,10 +655,47 @@ private void closeSearchFragment(@NonNull SearchFragment fragment) {
         getChildFragmentManager().beginTransaction().remove(fragment).commitNowAllowingStateLoss();
     }
 
+    public void switchToSearchFragment(ImageSearchFragment fragment,
+                                       @NonNull SearchInvokeSource source, @Nullable String query){
+        closeImageSearchFragment(fragment);
+        openSearchFragment(source, query);
+    }
+
+    @SuppressLint("CommitTransaction")
+    public void closeImageSearchFragment(@NonNull ImageSearchFragment fragment) {
+        getChildFragmentManager()
+                .beginTransaction().remove(fragment).commitNowAllowingStateLoss();
+    }
+
+    @SuppressLint("CommitTransaction")
+    private void openImageSearchFragment(List <String> labels) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1541123)